.github/workflows/test-types.yml

```diff
@@ -0,0 +1,16 @@
+name: Test Type Definitions
+on: pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "12"
```

Comment:
Isn't 12 a bit old at this point?

Reply:
I think I chose 12 because the Babel config specifies version <=12, but it could be made more re ent